src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimatorIO.h

```diff
@@ -59,6 +65,13 @@ struct Measurements
     Eigen::VectorXd encoders, encodersSpeed; /**< Joint position and joint velocity measurements */
     bool lfInContact{false}; /**< left foot contact state */
     bool rfInContact{false}; /**< right foot contact state */
+    
+    /** stamped contact status, 
+     * the usage of this map must be in a way 
+     * that every time an element is used, 
+     * it must be erased from the map 
+     */
+    std::map<int, std::pair<double, bool> > stampedContactsStatus;
```

Comment:
Niptick: pair may be difficult to read, I don't remember if we had the same problem somewhere else. Anyhow, this is not blocking.

Reply:
Yes, we had similar problem in SchmittTriggerDetector. I will fix it.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/151/commits/38f0af9f9e88504867ac9f9536a1697c145271cc